packages/plugins/plugin-inbox/src/components/Calendar/EventtList.tsx

```diff
@@ -23,7 +21,8 @@ export const EventList = ({ events = [], selected, onSelect }: EventListProps) =
         {events.map((event) => (
           <ListItem.Root
             key={event.id}
-            classNames={mx('flex flex-col cursor-pointer', ghostHover, selected === event.id && styles.selected)}
+            // TODO(burdon): Change bg-activeSurface to use data-active
```

Comment:
We can just `data-[active="true"]:bg-activeSurface`, conditionally applying tokens on `data-` and `aria-` attributes is already common throughout the repo.

If there are multiple places we can already identify should need this, we could also mint a semantic token which has `data-active` as a condition, but that means minting a new name to remember and use.